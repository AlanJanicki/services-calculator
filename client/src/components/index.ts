export { Card } from './Card/Card';
export { Checkbox } from './Checkbox/Checkbox';
export { Select } from './Select/Select';
export { Spinner } from './Spinner/Spinner';
