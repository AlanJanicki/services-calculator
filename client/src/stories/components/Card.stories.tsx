import { Card } from '../../components';

export default {
  component: Card,
  tags: ['autodocs'],
  title: 'Card'
};

export const Default = {
  args: {
    header: 'Kalkulator us≈Çug'
  }
};
