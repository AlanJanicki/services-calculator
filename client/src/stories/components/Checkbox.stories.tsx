import { Checkbox } from '../../components';

export default {
  component: Checkbox,
  tags: ['autodocs'],
  title: 'Checkbox'
};

export const Default = {
  args: {
    name: 'Internet'
  }
};
