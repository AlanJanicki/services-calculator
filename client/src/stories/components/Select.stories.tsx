import { Select } from '../../components';
import priceList from '../../mocks/servicesPriceList.json';

export default {
  component: Select,
  tags: ['autodocs'],
  title: 'Select'
};

export const Default = {
  args: {
    label: 'Wybierz rok',
    options: priceList.years
  }
};
