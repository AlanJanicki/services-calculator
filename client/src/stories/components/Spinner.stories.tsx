import { Spinner } from '../../components';

export default {
  component: Spinner,
  tags: ['autodocs'],
  title: 'Spinner'
};

export const Default = {};
