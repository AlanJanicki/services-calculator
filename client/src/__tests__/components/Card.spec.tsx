import React from 'react';

import { Card } from '../../components';
import { renderWithSnapshot } from '../common';

describe('<Card/>', () => {
  it('matches snapshot', () => renderWithSnapshot(<Card header='Kalkulator usÅ‚ug' />));
});
