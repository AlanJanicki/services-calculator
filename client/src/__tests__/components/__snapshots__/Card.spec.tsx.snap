// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> matches snapshot 1`] = `
<DocumentFragment>
  .c0 {
  margin: 0 auto;
  width: 80%;
  border: 1px solid #dfdfdf;
  border-radius: .25rem;
}

.c1 {
  height: 3rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #f7f7f7;
}

.c2 {
  padding: 1rem;
  display: grid;
}

@media (min-width:576px) {
  .c1 {
    height: 3.5rem;
  }
}

@media (min-width:576px) {
  .c1 h1 {
    font-size: 1.1rem;
  }
}

@media (min-width:768px) {
  .c1 h1 {
    font-size: 1.2rem;
  }
}

@media (min-width:992px) {
  .c1 h1 {
    font-size: 1.5rem;
  }
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      <h1>
        Kalkulator us≈Çug
      </h1>
    </div>
    <div
      class="c2"
    />
  </div>
</DocumentFragment>
`;
